localstack-s3/upload.js

```diff
@@ -0,0 +1,50 @@
+const AWS = require("aws-sdk");
+// const fs = require('fs')
+
+const KEY_ID = "nxserverless"
+const SECRET_KEY = "nxserverless"
+
+const BUCKET_NAME = "nxboy";
+
+const s3 = new AWS.S3({
+  region: "us-east-1",
+  accessKeyId: KEY_ID,
+  secretAccessKey: SECRET_KEY,
+  endpoint: 'http://localhost:4566', // This is the localstack EDGE_PORT
+  s3ForcePathStyle: true,
+});
+
```

Comment:
very specific s3 bucket? what's the intention here?

Reply:
@harellevy just an example to create a localstack s3 bucket